internal/db/plan.go

```diff
@@ -16,19 +16,19 @@ const (
 func GetPlan(db *gorm.DB, planName string) (*model.Plan, error) {
 	wrapMsg := fmt.Sprintf("unable to look up plan name '%s'", planName)
 	var err error
-
-	plan := model.Plan{Name: planName}
+	var plan = model.Plan{}
 	err = db.
+		Where("name=?", planName).
 		Preload("PlanQuotaDefaults").
 		Preload("PlanQuotaDefaults.ResourceType").
-		First(&plan).Error
+		First(&plan).
+		Scan(&plan).Error
```

Comment:
I experimented with this a little. It appears that the call to `Scan` is not necessary because `First` will scan the data in for us. Something like this will work:

``` go
	db.
		Where("name = ?", planName).
		Preload("PlanQuotaDefaults").
		Preload("PlanQuotaDefaults.ResourceType").
		First(&plan).
		Error
```

It looks like adding an explicit condition is necessary. (I assume that it's because we're not searching based on the primary key because I haven't been able to break the primary key search by omitting an explicit condition.) There are several options mentioned in the documentation, but the one that you've chosen will work just fine. My only suggestion is to remove the call to `Scan`.